incuna-sass/mixins/_svg.sass

```diff
@@ -0,0 +1,12 @@
+//////////  S V G  M I X I N  //////////////////////////////
+// To use like you would a sprite with the added benefit that
+// it can be whatever size you like and will work perfectly
+// on retina screens
+@mixin svg($file, $width: 100%, $height: $width)
+    @include background(image-url('svg/#{$file}.svg'))
+    background:
+        repeat: no-repeat
+        size: contain
+    width: $width
+    height: $height
+    display: inline-block
```

Comment:
except no, because in your first example you wouldn't need to include the mixin again. It's svg dude, that's the point
```
@include svg(blah)
display: block
@include media(whatever)
    width: bigger
    height: bigger
```

I understand what you're arguing. I just disagree